src/wrap.c

```diff
@@ -2010,17 +2031,38 @@ inspectLib(struct dl_phdr_info *info, size_t size, void *data)
         uint64_t got_value = *((uint64_t *)got_addr);
         if (!fname || !got_value) continue;
 
+        char *file_from_got_value = osFileNameFromAddr(got_value);
```

Comment:
we are malloc'ing for every iteration in the loop. would be more performant to pass in a char array created outside the loop.
if you feel this is a concern, change it. if not, i'm fine with leaving it as it is. just thought i would raise it.

Reply:
yeah. good point. i left it using malloc thinking that other uses of osFileNameFromAddr() would benefit from the malloc and it is consistent with the way similar os funcs are done. 

hoping to leave it for now, just because of the testing that is needed.